src/pivot-table/components/cells/EmptyHeaderCell.tsx

```diff
@@ -20,7 +20,7 @@ const EmptyHeaderCell = ({ rowSpan, columnSpan }: Props) => {
       }}
       data-testid={testId}
     />
-  );
+  ) : null;
```

Comment:
Maybe more Imperative way in this case would apply nicely? or am i the only one who is a bit obsessive about  conditional rendering? 😅
```jsx
if(rowSpan == 0 && colSpan == 0) return null;

return (
   <div style={{...}} />
)
```

Reply:
I also find that easier to read @a-m-dev